openedx/features/caliper_tracking/tests/current/edx.course.enrollment.upgrade.clicked.json

```diff
@@ -0,0 +1,21 @@
+{
+    "username":"honor",
+    "event_source":"browser",
+    "name":"edx.course.enrollment.upgrade.clicked",
+    "accept_language":"en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
+    "time":"2018-10-17T13:08:35.005410+00:00",
+    "agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+    "page":"https://tasawernawaz.sandbox.edx.org/dashboard",
+    "host":"tasawernawaz.sandbox.edx.org",
+    "session":"f3029807d5ac0b71b59b53e9e789f0fc",
+    "referer":"https://tasawernawaz.sandbox.edx.org/dashboard",
+    "context":{
+        "user_id":7,
```

Comment:
The `user_id` should be same in both files. It is 6 in one while 7 in the other.

Reply:
Fixed